src/foam/u2/view/MultiSelectView.js

```diff
@@ -0,0 +1,186 @@
+/**
+ * @license
+ * Copyright 2025 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+
+foam.CLASS({
+
+  package: 'foam.u2.view',
+  name: 'MultiSelectView',
+  extends: 'foam.u2.View',
+
+  documentation: `
+      The choices are in [value, label, isSelected ] quartets.
+
+      However the client can simply pass in [value, label] and it will adapt to a [value, label, isSelected, choiceMode ] format
+  `,
+
+  css: `
+    ^ .foam-u2-ActionView-options {
+      padding: 10px;
+      border: 1px solid $grey400;
+      cursor: pointer;
+      min-width: 400px;
+      display: flex;
+      flex-direction: row-reverse;
+      align-items: center;
+      justify-content: space-between;
+      background-color: $white;
+      color: $black;
+      border-radius: 4px;
+  }
+  ^ .foam-u2-ActionView-options:hover {
+    background-color: $white !important;
+    color: $black !important;
+  }
+  ^choices-holder {
+      background: $white!important;
+      color: $black !important;
+      opacity: 1 !important;
```

Comment:
Not a fan of all these !importants, these should be scoped proeprly if overrides are not working